### PR TITLE
MAINT: The crackfortran function is called with an extra argument

### PR DESCRIPTION
**Pull request by ceh**
See https://github.com/numpy/numpy/blob/a000144df/numpy/f2py/crackfortran.py#L3253 for the function signature.

### CODE REVIEW

File: numpy/f2py/crackfortran.py [hunk 1]
```diff
@@ -3341,7 +3341,7 @@ def crack2fortran(block):
   and also be sure that the files do not contain programs without program statement).
 """, 0)
 
-    postlist = crackfortran(files, funcs)
+    postlist = crackfortran(files)
```
Comment (charris) [inline numpy/f2py/crackfortran.py]: The module documentation fot `crackfortran` also needs fixing.

**Comment by charris** [comment]
Going to close and reopen as experiment.

File: numpy/f2py/crackfortran.py (hunk 1 above)
Comment (ceh) [reply numpy/f2py/crackfortran.py]: Updated.

**Comment by charris** [comment]
Thanks @ceh .

**Pull request merged**